packages/cms-data-sync/src/index.ts

```diff
@@ -1,8 +1,33 @@
 /* istanbul ignore file */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { createClient } from 'contentful-management';
 import { migrateExternalAuthors } from './external-authors/external-authors.data-migration';
 import { migrateTeams } from './teams/teams.data-migration';
+import { migrateCalendars } from './calendars/calendars.data-migration';
 
 (async () => {
+  const {
+    CONTENTFUL_MANAGEMENT_ACCESS_TOKEN,
+    CONTENTFUL_SPACE_ID,
+    CONTENTFUL_ENV_ID,
+  } = process.env;
+
+  const contentfulClient = createClient({
+    accessToken: CONTENTFUL_MANAGEMENT_ACCESS_TOKEN!,
+  });
+
+  const contentfulSpace = await contentfulClient.getSpace(CONTENTFUL_SPACE_ID!);
+  const webhook = await contentfulSpace.getWebhook(
+    `${CONTENTFUL_ENV_ID!.toLowerCase()}-webhook`,
+  );
+
+  webhook.active = false;
+  await webhook.update();
+
   await migrateTeams();
```

Comment:
i don't think this file is tested but it might be worth adding some tests now since the logic is becoming a little bit more complex... 

anyway the reason i bring it up is because I think in the previous review of this I asked what would happen if one of the migrations failed after you disabled a webhook - looking at the code I think it would never activate the webhook back, but maybe you decided not do do it because technically the individual migrations should not throw - I still think something could go wrong and we definitely don't want to leave those webhooks inactive so you may want to add something here for that.

Reply:
Sorry, I've missed the idea of the other comment. I've added a try catch finally block now